components/searchable_channel_list.jsx

```diff
@@ -193,7 +207,16 @@ export default class SearchableChannelList extends React.Component {
                             onInput={this.doSearch}
                         />
                     </div>
+                    <div className='create_button'>
+                        {this.props.createChannelButton}
```

Comment:
Having the Create New Channel button next to the search box feels weird to me like the it should somehow be connected to what I type in the search box. Any thoughts, @esethna?
![image](https://user-images.githubusercontent.com/3277310/47751233-9a1af400-dc67-11e8-867b-d396b14cc91b.png)

In terms of the code, I feel like this might be coupling this list to the modal a bit much, but I don't have a strong argument for actually changing it since it's already pretty specific to this use case.

Reply:
I can't find the designs given to me by MM but if I'm correct this is how it was decided.  @esethna or @asaadmahmood maybe you can provide the design mockups for the sidebar work.